source/_integrations/timer.markdown

```diff
@@ -54,6 +54,15 @@ timer:
       description: Set a custom icon for the state card.
       required: false
       type: icon
+    restore:
+      description: When true, active timer and paused timers will be restored on startup. If an active timer was supposed to end while Home Assistant is stopped, the `restore_grace_period` property controls which event fires.
+      required: false
+      type: boolean
+      default: false
+    restore_grace_period:
```

Comment:
Should we just always fire when the restore option is enabled and make sure we can access the original finish time (e.g., by including it in the event?)